### PR TITLE
Enable to use a specific branch

### PR DESCRIPTION
**Pull request by pixyzehn**
[Suggestion]
I'd like to use a specific branch(like swift4 branch in near feature) not only version or `master` branch.
What do you think that? If you agree, should I include master in new branch feature?
@JohnSundell 

For instance,
```sh
$ TestDrive Unbox -b swift3
🕵️‍♀️  Finding pod 'Unbox'...
📦  Cloning https://github.com/johnsundell/unbox.git...
📋  Checking out swift3...
🚗  Unbox is ready for test drive

⚡️  Generating workspace at /Users/pixyzehn/Dropbox/Developer/projects/TestDrive/TestDrive-Unbox.xcworkspace/...

🚘  Test driving Unbox
```

If there is no branch,
```sh
$ TestDrive Unbox -b hoge
  Finding pod 'Unbox'...
📦  Cloning https://github.com/johnsundell/unbox.git...
📋  Checking out hoge...

💥  ShellOutError(message: "error: pathspec \'hoge\' did not match any file(s) known to git.", output: "")
```

### CODE REVIEW

**Comment by pixyzehn** [comment]
@JohnSundell Good idea! I'll update them:)

**Comment by pixyzehn** [comment]
@JohnSundell Thank you for your advice! I just updated👍

File: Sources/TestDrive.swift [hunk 1]
```diff
@@ -27,20 +27,19 @@ extension CommandLine {
...
             }
 
             switch argument {
             case "--platform", "-p":
                 expectingPlatform = true
-            case "--version", "-v":
-                expectingVersion = true
             case "--master", "-m":
                 parsedArguments.addTagToLastTarget(.master)
+            case "--checkout", "-c":
```
Comment (JohnSundell) [inline Sources/TestDrive.swift]: I think we should keep this `--version`/`-v`. No need to break the API if don't need to, and I think even for branches it makes sense to treat it as a "version" publically, even if we internally refer to it as a "checkout".

Comment (pixyzehn) [inline Sources/TestDrive.swift]: I agree with you. I'll modify later. Thanks!

**Comment by pixyzehn** [comment]
@JohnSundell I avoided breaking API changes. Please check it :)

**Comment by pixyzehn** [comment]
Thanks too😉

**Pull request merged**